File: NapakalakiRuby/lib/death_bad_consequence.rb
#encoding: utf-8
#Autor: Dunia Rodríguez y Emilio Reguero
module NapakalakiGame
class DeathBadConsequence < NumericBadConsequence
 
  attr_reader :death  
  
  def initialize(text)
    @death = true
    super(text, Player.getMaxLevel, @@MAXTREASURES, @@MAXTREASURES)
  end
  
  def is_empty
    result = false
  end
  
  def to_s
    respuesta = super.to_s + "\n Has muerto"
  end
  end
end

File: Napakalaki/src/NapakalakiGame/DeathBadConsequence.java
/*
* PDOO Practicas - Napakalaki Java
* @author Emilio Reguero y Dunia Rodríguez 
*/
package NapakalakiGame;


public class DeathBadConsequence extends NumericBadConsequence {
    
    
    private boolean death;
    
    public DeathBadConsequence (String text, boolean death){
        super(text,Player.MAXLEVEL,MAXTREASURES,MAXTREASURES);
        this.death = death;
    }
    
    @Override
    public boolean isEmpty ()
    {
        return false;
    }
    
    @Override
    public String toString(){
       return "You are dead.";
    }
}
